src/Npgsql.GeoJSON/CrsMap.WellKnown.cs

```diff
@@ -0,0 +1,585 @@
+﻿namespace Npgsql.GeoJSON
+{
+    partial struct CrsMap
+    {
+        internal static readonly CrsMapEntry[] s_wellKnown =
```

Comment:
Also rename `s_wellKnown` to `WellKnown` to match static readonly naming convention.

Reply:
Okay.